build.gradle.kts

```diff
@@ -40,7 +40,7 @@ dependencies {
 
 allprojects {
     group = "org.kodein.internal.gradle"
-    version = "2.6.0"
+    version = "2.6.1"
```

Comment:
This is a rather important feature, not a bugfix or a minor improvement.
I'd go `2.7.0`.